src/pages/state/[state].js

```diff
@@ -1,284 +1,284 @@
-              <ul>
-                {notes?.map((note, i) => (
-                  <li key={`${name}-note-${i}`}>
-                    <ReactMarkdown>{note}</ReactMarkdown>
-                  </li>
-                ))}
-              </ul>
-            </Row>
-          ))} */}
-      {stateData.videos?.length || stateData.voicesForReform?.length ? (
-        <Row className={styles.voices}>
-          <Row>
-            <h3>Voices For Reform</h3>
-          </Row>
-          {stateData.voicesForReform?.length ? (
-            <Row>
-              <ul>
-                {stateData.voicesForReform.map(({ note }, i) => (
-                  <li key={"vfr-notes-" + i}>
-                    <ReactMarkdown>{note}</ReactMarkdown>
-                  </li>
-                ))}
-              </ul>
-            </Row>
-          ) : null}
-          {stateData.videos?.length
-            ? stateData.videos.map(({ link, title, description }, i) => (
-                <Row key={`statewide-videos-${i}`} className={styles.video}>
-                  <ReactPlayer url={link} width={594} height={361} />
-                  <h4>{title}</h4>
-                  {description && (
-                    <ReactMarkdown className={styles.description}>
-                      {description}
-                    </ReactMarkdown>
-                  )}
-                </Row>
-              ))
-            : null}
-        </Row>
-      ) : null}
-      {outOfStateOpos.length > 0 && (
-        <Row className={styles.serviceTable}>
-          <OpoTable
-            headings={headings}
-            inState={false}
-            opos={outOfStateOpos}
-            title="OPO Performance in Neighboring States"
-          />
-        </Row>
-      )}
-    </Layout>
-  );
-}
-
-export const query = graphql`
-  query {
-    statesGeoData: file(relativePath: { eq: "data/states.geojson" }) {
-      childGeoJson {
-        features {
-          geometry {
-            type
-            coordinates
-          }
-          properties {
-            abbreviation
-          }
-          type
-        }
-      }
-    }
-  }
-`;
+import React from "react";
+import { graphql, navigate } from "gatsby";
+import { Col, Row } from "react-bootstrap";
+import ReactMarkdown from "react-markdown";
+import ReactPlayer from "react-player";
+import booleanIntersects from "@turf/boolean-intersects";
+
+import Layout from "../../components/layout/layout";
+import Map from "../../components/map/map";
+import SelectState from "../../components/selectState/selectState";
+import OpoTable from "../../components/opoTable/opoTable";
+
+import {
+  findStateFeature,
+  formatStateName,
+  formatNumber,
+} from "../../utils/utils";
+import useDataMaps from "../../hooks/useDataMaps";
+import content from "./[state].content.yml";
+
+import * as styles from "./state.module.css";
+
+export default function State({ data: { statesGeoData }, state }) {
+  const [{ opoDataMap, stateDataMap }] = useDataMaps();
+

```

Comment:
This file was a diff when I pushed this PR, I had it accept the v2 changes so the file was changed to reflect the v2 version